### ISSUE TITLE
Test deeper architecture

### ISSUE THREAD

**Issue by bnord** [GeorgiadouAntigoni/weak_lensing_machine_learning#5]
Test a deep convolutional network with ReLU and same number of neurons as in Arch0Dense from #2

**Comment by GeorgiadouAntigoni** [comment]
![gridtests_cnn_alp5-seed10-epochs500](https://user-images.githubusercontent.com/21352033/53505476-85196a80-3a79-11e9-8a46-e3a42bb20549.png)

I have been testing a deep CNN with ReLU with the same architecture as Arch0Dense. This is a plot of the sigmas versus the number of neurons of the layers alp =5, # of seeds =10 and epochs = 500 . I am running for more seed options (100) but the colab keeps stopping the process, so I have to restart. Can we run in Wilson cluster?

**Comment by bnord** [comment]
yep, we can run on wilson. We'll need to get you an account. Can you email Amitoj G Singh <amitoj@fnal.gov> to ask for an account?

**Comment by bnord** [comment]
Why does colab stop the process? 
Is it after a certain amount of time?

**Comment by bnord** [comment]
There’s also a wall time on Wilson. We'll want to know what that is. We might need to save the network and re-start whether on Wilson or on colab.

**Comment by GeorgiadouAntigoni** [comment]
I noticed that Colab stops before the 12 hour limit. I think it is not intended for long-running tasks. 

From the Colab FAQs web page:

"Colaboratory is intended for interactive use. Long-running background computations, particularly on GPUs, may be stopped. Please do not use Colaboratory for cryptocurrency mining. Doing so is unsupported and may result in service unavailability. We encourage users who wish to run continuous or long-running computations through Colaboratory’s UI to use a local runtime."

I save the instances in .h5 files though, so it doesn't start from scratch.

**Comment by bnord** [comment]
Should we try wilson?

**Comment by GeorgiadouAntigoni** [comment]
Yes, let's try. I'll ask for an account.
